### DISCUSSION TITLE
开源软件镜像站有哪些？

### DISCUSSION THREAD

**Discussion by v5tech** [alterem/daily-poetry-image#2] [Q&A] [answered]
开源软件镜像站有哪些？

**Comment by alterem** [comment, marked as answer]
中国高校镜像站
https://mirrors.sjtug.sjtu.edu.cn/
https://mirrors.bfsu.edu.cn/
https://mirrors.zju.edu.cn/
https://mirrors.ustc.edu.cn/
https://mirrors.tuna.tsinghua.edu.cn/
https://mirrors.bupt.edu.cn/
